### DISCUSSION TITLE
Water temperature Set point Cooling

### DISCUSSION THREAD

**Discussion by tosomax** [Trunks1982/Daikin-Rotex-HPSU-CAN#46] [Ideas]
Hi

Would it be possible to configure the setpoint of the water in cooling mode, when working with a fixed temperature?
I'm referring to this parameter:
![image](https://github.com/Trunks1982/Daikin-Rotex-HPSU-CAN/assets/155764036/07af718b-b73d-47b3-b539-cdeff8fb90ba)
I don't know what would it be in deutsche

I think this could be helpful during the summer to regulate the water temperature based on the external temperature and the time of the day through home assistant.

Thank you for keeping this into consideration :)

**Comment by Trunks1982** [comment]
Ja ist möglich.😉
Aber eigentlich stellt man das einmal alles auf 18 grad und belässt es so.

**Reply by tosomax** [reply]
If you also think this is a good idea and want to implement it, I would appreciate it

Of course no rush, it would just be nice.

In case you don't want to do it, if you had the CAN parameters I could try to do it by myself taking inspiration from your code :)

By the way, since I upgraded the code to the latest version I never had any more errors in the heat pump, so thanks again for the great work!

**Reply by Trunks1982** [reply]
Ich kann ja mal gucken wann ich zeit dafür finde.
Sind dann nämlich mehrere Werte die man dann verstellen kann.min vl kühlen usw

**Reply by Trunks1982** [reply]
Läuft denn sonst alles soweit?

**Reply by tosomax** [reply]
Yeah don't rush it, I'm just giving an idea, I'm not expecting you to do it just cause I asked it :)

I don't have any more errors on the heat pump. However, I'm seeing that after a few days of uptime the ESP32 starts to be unavailable every 10-15 minutes and goes back online, then suddenly it is no reachable anymore. I can't tell if it is crashing or just loosing the WiFi connection. I will try first to connect it to a wifi repeater which is near to the heat pump, and if this doesn't work I have two new ESP32 to flash and try in case this one is defective (these are cheap aliexpress ESP32)

I don't believe the issue I'm having is due to your code.

**Reply by Trunks1982** [reply]
Ich glaube auch das es auf eine schlechte wlan verbindung zurück zuführen ist.
